.github/workflows/flowzone.yml

```diff
@@ -1199,6 +1306,77 @@ jobs:
           versionbot: false # disable the included versionbot branch checkout
           source: ${{ inputs.working_directory }}
 
+  ###################################################
+  ## Python
+  ###################################################
+
+  python_test:
```

Comment:
You should make sure any steps that need to depend on `python_test`, at the very least the `protect_branch` job should depend on it